docs/pages/ios-quickstart.md

```diff
@@ -7,147 +7,39 @@ title: FlowForms iOS quickstart
-bind(
-    fieldValue field: String,
-    id: String,
-    completion: @escaping (String) -> Void
-) -> Binding<`String`>
-</code></pre>
-
-<p class="comment"> The implementation will be: </p>
+**2 :** Create an instance of **SignupViewModel** within our **FormState** or class that stores the state of the form:
 
 <pre><code class="swift">
-class FormState: ObservableObject {
-    let formModel = FormModel()
-    
-    var name: Binding<`String`> {
-        formModel.form.bind(
-            fieldValue: formModel.username,
-            id: FormModel.companion.USERNAME
-        ) {
-            self.formModel.username = $0
-            self.objectWillChange.send()
-        }
-    }
-    
-    var password: Binding<`String`> {
-        formModel.form.bind(
-            fieldValue: formModel.password,
-            id: FormModel.companion.PASSWORD
-        ) {
-            self.formModel.password = $0
-            self.objectWillChange.send()
-        }
-    }
-}
-</code></pre>
-
-<p class="comment"> In the View we can respond to those Bindings in this way: </p>
+import shared
+import SwiftUI
 
-<pre><code class="swift">
-struct FormView: View {
-  @ObservedObject var formState: FormState
-	
-    var body: some View {
-        VStack {
-            TextField("Username textfield", text: formState.username)
-            SecureField("Password textfield", text: formState.password)
-        }
-    }
-}
-</code></pre>
-
-<p class="comment"> In Kotlin, we are getting each field using the field(...) function and then collecting (observing) their status. We display an error message in their input layouts when the field is incorrect (i.e. a field's validation failed). Both REQUIRED_UNSATISFIED and MIN_LENGTH_UNSATISFIED are specific status codes of the defined fields' validations in the form declaration (on the VM). Additionally, we are collecting the general form status, enabling a "continue" button when the form is correct (i.e. all its fields are correct) and disabling it when any of its fields are incorrect (at least one validation failed on some field).</p>
-
-
-**3 :** In swift we can declare **@Published** variables to publish the statuses from each field, i.e:
+final class FormState: ObservableObject {
+  let viewModel = SignupViewModel()
 
-<pre><code class="swift">
-class FormState: ObservableObject {
-	//...
-	@Published var usernameStatus: String = StatusCodes.shared.UNMODIFIED
-    @Published var passwordStatus: String = StatusCodes.shared.UNMODIFIED
-    @Published var formStatus: String = StatusCodes.shared.UNMODIFIED
-	//...
+  init() { }
 }
 </code></pre>
 
-<p>To observe any errors or statuses provided by Swift Publisher, we need to bind them with the field we receive from the formModel. The iOS library source code includes a tool that can be used to bind the publishers with the statuses.</p>
+<p class="comment"> SignupViewModel is located within the common module and its pure kotlin code so it can be used by both platforms</p>
```

Comment:
```suggestion
<p class="comment">Since SignupViewModel is located within the common module and it is pure kotlin code we can use it directly in iOS</p>
```